lte/cloud/go/services/lte/servicers/builder_servicer_test.go

```diff
@@ -1421,6 +1465,10 @@ func TestBuilder_Build_MMEPool(t *testing.T) {
 			},
 			HeConfig: &lte_mconfig.PipelineD_HEConfig{},
 			LiUes:    &lte_mconfig.PipelineD_LiUes{},
+			FeatureSet5G: &lte_mconfig.PipelineD_FeatureSet5G{
+				Enable:         false,
+				NodeIdentifier: "192.168.200.1",
+			},
```

Comment:
please go over the test and see that you do need to add an assert to check the result is what you expected. Otherwise this is not a test.

Please use intellij as a IDE to run the test locally and understand how valuable those test are and how powerful the debugger it is.

Reply:
Sure, thanks for suggestion.